client/src/components/UserGuideButton.vue

```diff
@@ -1,15 +1,89 @@
+<script>
+import UserGuideDialog from '@/components/UserGuideDialog.vue';
+
+export default {
+  components: {
+    UserGuideDialog,
+  },
+  props: {
+    annotating: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data() {
+    return {
+      dialog: false,
+      userGuideLink: 'https://github.com/VIAME/VIAME-Web/wiki/User-Documentation',
+    };
+  },
+};
+</script>
 <template>
-  <v-btn
-    dense
-    depressed
-    href="https://github.com/VIAME/VIAME-Web/wiki/User-Documentation"
-    target="_blank"
-    color="secondary darken-2"
-    class="mx-3"
-  >
-    User Guide
-    <v-icon class="pl-2">
-      mdi-help-circle
-    </v-icon>
-  </v-btn>
+  <div>
+    <v-btn
+      v-if="!annotating"
+      dense
+      depressed
+      :href="userGuideLink"
+      target="_blank"
+      color="secondary darken-2"
+      class="mx-3"
+    >
+      User Guide
+      <v-icon class="pl-2">
+        mdi-help-circle
+      </v-icon>
+    </v-btn>
+    <v-dialog
+      v-else
+      v-model="dialog"
+      width="600"
+    >
+      <template #activator="{on}">
+        <v-btn
+          dense
+          depressed
+          target="_blank"
+          color="secondary darken-2"
+          class="mx-3"
+          v-on="on"
+        >
+          Help
+          <v-icon class="pl-2">
+            mdi-help-circle
+          </v-icon>
+        </v-btn>
+      </template>
+      <v-card>
+        <v-card-title>
+          Help
+        </v-card-title>
+        <v-card-text>
+          <user-guide-dialog />
+          <v-card-text>
```

Comment:
I don't think having nested v-card-text here is necessary, worked fine for me without.

Reply:
It added some minor padding around the user guide button, but it can be removed.